src/main/java/org/jabref/logic/citationkeypattern/BracketedPattern.java

```diff
@@ -868,14 +868,6 @@ public static String authorsAlpha(AuthorList authorList) {
         return alphaStyle.toString();
     }
 
-    static String getLastName(Author author) {
-        String[] nameParts = author.getNamePrefixAndFamilyName()
-                                   .replaceAll("\\s+", " ")
-                                   .trim()
-                                   .split(" ");
-        return nameParts[nameParts.length - 1];
-    }
```

Comment:
Code hint - if you encounter a similar case in the future, there is a more intuitive code IMHO:

https://www.w3schools.com/java/ref_string_lastindexof.asp

```java
int pos = string.lastIndexOf(" ");
String result = string.substr(pos +1);
```